static/dependencies.conf.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+use Dice\Dice;
+use Friendica\App;
+use Friendica\Core\Config;
+use Friendica\Factory;
+use Friendica\Util;
+use Psr\Log\LoggerInterface;
+
+return [
```

Comment:
We'll need some serious comments here to explain each individual rule. Not only it can save someone the need to read [the entire Dice documentation](https://r.je/dice), but it'll also help adding new rules for similar use cases.

Reply:
Yep, you're right!
As far as I can see, there shouldn't be THAT complex configuration after the initial core classes needed. Most of the others would be loaded without additional config.